### DISCUSSION TITLE
Registration username blacklist?

### DISCUSSION THREAD

**Discussion by fedi-will** [mastodon/mastodon#22753] [Q&A] [answered]
On my xmpp server I implemented a username blacklist, so others cannot register the name administrator@mydomain.com, for example. 

Is there some kind of way to do this with mastodon?
(Would you consider this a useful function?)

**Comment by connyduck** [comment, marked as answer]
There is a list of reserved names in `config/settings.yml` you can edit that for your server.

**Reply by fedi-will** [reply]
Ah, "reserverd_usernames:"
Exactly what I was looking for. 
Thank you so much @connyduck

**Comment by sepTN** [comment]
Is there an option to limit the minimum length of a username?
